src/main/java/uk/gov/gchq/gaffer/doc/function/FunctionChainExample.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.doc.function;
+
+import uk.gov.gchq.koryphe.impl.function.FunctionChain;
+import uk.gov.gchq.koryphe.impl.function.ToSet;
+import uk.gov.gchq.koryphe.impl.function.ToUpperCase;
+import uk.gov.gchq.koryphe.tuple.ArrayTuple;
+
+public class FunctionChainExample extends FunctionExample {
+    public static void main(final String[] args) {
+        new FunctionChainExample().runAndPrint();
+    }
+
+    public FunctionChainExample() {
+        super(uk.gov.gchq.koryphe.impl.function.FunctionChain.class);
+    }
+
+    @Override
+    protected void runExamples() {
+        functionChain();
+    }
+
+    private void functionChain() {
```

Comment:
This is quite a useful function so some more examples with different functions and inputs would be good